kfs-core/src/main/resources/edu/arizona/kfs/fp/document/validation/configuration/DisbursementVoucherValidation.xml

```diff
@@ -11,6 +11,12 @@
                            http://www.springframework.org/schema/aop

```

Comment:
This override belongs in edu/arizona/kfs/fp/document/validation/configuration/FinancialProcessingValidators.xml. The edu/arizona/kfs/fp/document/validation/configuration/DisbursementVoucherValidation.xml file should only contain beans related to document events, addAccountingLineValidation, updateAccountingLineValidation, routeDocumentValidation. etc.